webapp/templates/jury/clarification.html.twig

```diff
@@ -82,9 +82,9 @@ Jury
 
 <div class="col-sm">Queue: <span class="clarification-queue">{{ clar.queue }}
 <button class="btn btn-sm btn-link clarification-queue-change-button"><i class="far fa-edit" title="Change queue"></i></button></span>
-<span class="clarification-queue-form" data-current-selected-queue="{{ clar.queueid }}" data-clarification-id="{{clar.clarid}}">
+<div style="display: inline" class="clarification-queue-form" data-current-selected-queue="{{ clar.queueid }}" data-clarification-id="{{clar.clarid}}">
```

Comment:
Is it necessary to add inline styles, or can we do this in the css?

Reply:
We can do this in the CSS, my intention was to first get W3c compliant and when this is done move it to the CSS files. We have this on more locations still.